src/css/components/_live.scss

```diff
@@ -1,11 +1,22 @@
-.video-js.vjs-live .vjs-time-control,
-.video-js.vjs-live .vjs-time-divider,
-.video-js.vjs-live .vjs-progress-control {
-  display: none;
+.vjs-live-control {
+  display: block;
+  font-size: 1em;
+  line-height: 3em;
 }
 
-.video-js .vjs-live-control {
+.vjs-live .vjs-time-control,
+.vjs-live .vjs-time-divider,
+.vjs-live .vjs-progress-control {
```

Comment:
If you mean have the live component manage the visibility of the other components, I think that would become a mess. I did consider having each of those components hide itself when the player is live, but I'm on the fence and not sure where to draw the line of when we use CSS states and when we use javascript for hiding. I'm also not sure how Live+DVR will affect these, so for now I think I'd lean towards leaving them like this.